keep codeowners and templates in lockstep w/ paar (#246)
CrunchyData/pgmonitor: .github/pull_request_template.md

- [ ] Documentation update only  

# How Has This Been Tested?  
**Tested Configuration**:  
- [ ] Ansible, Specify version(s):  
- [ ] Specify Distro & version(s):  
- [ ] Specify Repo Distro & version(s):  

> - [ ] PostgreQL, Specify version(s):  

Reviewer: ...and fix the original paar one?

> - [ ] PostgreQL, Specify version(s):  

Reviewer: Also, do we need to do this for paap too?

> - [ ] PostgreQL, Specify version(s):  

Author: > Can we fix the PostgreSQL typo since we're already syncing things?

fixed

> - [ ] PostgreQL, Specify version(s):  

Author: > ...and fix the original paar one?

see #1050 in paar

> - [ ] PostgreQL, Specify version(s):  

Author: > Also, do we need to do this for paap too?

i dunno. @sharmay ?

> - [ ] PostgreQL, Specify version(s):  

Reviewer 2: not for paap

> - [ ] Specify Repo Distro & version(s):  

Reviewer 3: What does repo distro/version mean in context with pgMonitor?

Author: I assume the same as w/ paar. Are you using the CentOS rpms from us or the RHEL rpms from us. Which probably doesn't make a lot of sense, since they could be using upstream source, yes?

Reviewer 3: Ahh ok. Thought asking the Distro would answer this question, but forgot they could possibly be using centos repo on rhel or vice versa. Yes, they could possibly be using upstream for Prometheus/Grafana. But I guess we'd want to know that as well. 

Maybe clarify that you mean package repository?

Author: how's that @keithf4 ?

Reviewer 3: Looks good

Author: should i do the same for paar then? i think 'yes'

Reviewer: "Packages compiled from source" suggests to me something different than I think you're intending, and is at least ambiguous.  Rephrase?

Author: what do you think it meand and what do you think I intend @pgguru ?

Reviewer: @hunleyd see my suggestions for reworking this area to get the idea of how it might be disambiguated.

- [ ] Documentation update only  

# How Has This Been Tested?  
**Tested Configuration**:  
- [ ] Ansible, Specify version(s):  
- [ ] Specify Distro & version(s):  

> - [ ] Specify Package distro & version(s):  
> - [ ] Packages compiled from source & their version:  

Reviewer: ```suggestion
- Installation method:  
  - [ ] Binary install from source, version:  
  - [ ] OS package repository; distro and version:  
  - [ ] Local package server, version:  
  - [ ] Custom-built packages, version:  
  - [ ] Other:  
```

- [ ] PostgreSQL, Specify version(s):  
- [ ] docs tested with hugo version(s):  

> Tested with playbook(s):  

Reviewer: ```suggestion
Tested with playbook(s):  

```

This tends to be crowded up with the next section and more than once I missed this section early on.

# How Has This Been Tested?  
**Tested Configuration**:  
- [ ] Ansible, Specify version(s):  
- Installation method:  
    - [ ] Binary install from source, version:  

>     - [ ] OS ackage repository, distro and version:  

Reviewer: ```suggestion
    - [ ] OS package repository, distro and version:  
```

>     - [ ] Local package session, version:  

Reviewer: ```suggestion
    - [ ] Local package server, version:  
```

…am I off base? What’s a local package session? I was thinking like a custom package that was built internally and distributed using the customer’s own infrastructure.

Reviewer: Or maybe that’s still covered by the custom package option. @gregscds ?

Reviewer 3: Yeah I'm confused by this one as well. Remember this is a public repo so we have to make sure the things in this template would make sense to almost anyone coming in and opening in an issue.
